Doc/c-api/dict.rst

```diff
@@ -64,7 +64,8 @@ Dictionary Objects
 
    Insert *value* into the dictionary *p* with a key of *key*.  *key* must be
    :term:`hashable`; if it isn't, :exc:`TypeError` will be raised. Return
-   ``0`` on success or ``-1`` on failure.
+   ``0`` on success or ``-1`` on failure.  This function *does not* steal a
+   reference to *val*.
 
 
 .. c:function:: int PyDict_SetItemString(PyObject *p, const char *key, PyObject *val)
```

Comment:
I 'd like to suggest to change the signature into
```
 .. c:function:: int PyDict_SetItemString(PyObject *p, const char *key, PyObject *value) 
```

IMHO, this change makes documentation more consistent since the documentation say `Insert *value* into the dictionary`

The current implementation does not use val.
reference: https://github.com/python/cpython/blob/2d5bf568eaa5059402ccce9ba5a366986ba27c8a/Objects/dictobject.c#L3399

Reply:
This suggestion could be applied to `int PyDict_SetItem`

Reply:
@corona10  Nice catch :) . I made the docs consistent with the signature.